sklearn/calibration.py

```diff
@@ -496,11 +516,10 @@ def _fit_classifier_calibrator_pair(
     y,
     train,
     test,
-    supports_sw,
     method,
     classes,
     sample_weight=None,
-    **fit_params,
+    fit_params=None,
```

Comment:
was there a reason for this change?

Reply:
The call to [`_fit_classifier_calibrator_pair`](https://github.com/BenjaminBossan/scikit-learn/blob/c3a1d67df287569f0335bba9661c03d0fb77fc9c/sklearn/calibration.py#L373-L383) is now analogous to the call directly below to [`cross_val_predict`](https://github.com/BenjaminBossan/scikit-learn/blob/c3a1d67df287569f0335bba9661c03d0fb77fc9c/sklearn/calibration.py#L389-L398). I favored the consistency there but I can also revert if you want.

Reply:
That's fair!